src/OctoshiftCLI.Tests/gei/Handlers/MigrateRepoCommandHandlerTests.cs

```diff
@@ -325,10 +328,11 @@ public async Task Happy_Path_GithubSource_Ghes()
 
             var gitArchiveUrl = $"https://example.com/{gitArchiveId}";
             var metadataArchiveUrl = $"https://example.com/{metadataArchiveId}";
-            var gitArchiveContent = new byte[] { 1, 2, 3, 4, 5 };
-            var metadataArchiveContent = new byte[] { 6, 7, 8, 9, 10 };
+
```

Comment:
nit: remove extra empty line

Reply:
Addressed in 7d436e04071bf868990e1d32d6276668c1db1a35 :tada: